pkg/consts/doctype.go

```diff
@@ -142,4 +142,8 @@ const (
 	// NextCloudFiles doc type is used when listing files from a NextCloud via
 	// WebDAV.
 	NextCloudFiles = "io.cozy.remote.nextcloud.files"
+	// ChatCompletions doc type is used for a chat between the user and a chatbot.
+	ChatCompletions = "io.cozy.ai.chat.completions"
```

Comment:
I feel like it's more "conversation" than "completion" : completion sounds more like the API action, while the conversation is more the actual data type